deployment/environment.go

```diff
@@ -55,6 +59,11 @@ type Chain struct {
 	// Users are a set of keys that can be used to interact with the chain.
 	// These are distinct from the deployer key.
 	Users []*bind.TransactOpts
+
+	// ZK deployment specifics
+	IsZk          bool
+	ClientZk      *clients.Client
+	DeployerKeyZk *accounts.Wallet
```

Comment:
Same question here, these are all ZKSync specific and not ZK specific right?

Reply:
This is the same answer. The chains using this kind of EVM require specific methods to deploy. That is why we have a specific client too.

Reply:
The real name of the VM is Era VM that is very similar to EVM (confusing). We can use a different naming. I like `ZkEVM` word